scripts/qtcreator/templates/wizards/openframeworks/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include "ofMain.h"
```

Comment:
Have you had a chance to look this?  I am trying to get oF 0.9.0 running on OSX (works fine on Windows already).  Thanks.